tests/reconfiguration.py

```diff
@@ -723,6 +723,7 @@ def run_all(args):
         test_issue_fake_join(network, args)
 
         if args.consensus != "BFT":
+            test_add_as_many_pending_nodes(network, args)
```

Comment:
This is the real fix, I think. Where we were previously running this, we'd added enough nodes to reach a 7-node network. This test case then adds 7 more, meaning the test box is maintaining 14 nodes at once. Where 2tx reconfig is enabled, and this test is executed by both in parallel, we could end up with 28 nodes on a single box.

That leads to each node running much slower than our timeouts can handle, producing spurious TLS handshake failures, Raft elections, and eventual timeouts in the top-level testing asserts.